Sources/StreamChat/APIClient/Endpoints/Payloads/UserPayloads.swift

```diff
@@ -191,3 +191,59 @@ struct UserUpdateRequestBody: Encodable {
         try extraData?.encode(to: encoder)
     }
 }
+
+// MARK: - Current User Unreads
+
+struct CurrentUserUnreadsPayload: Decodable {
```

Comment:
All `var`s could be `let`s here but not hugely important

Reply:
Done !